### ISSUE TITLE
Only syncing 20 books. Same issue as numbers 191, 489, 492, 502, 503, and 507.

### ISSUE THREAD

**Issue by steve5053872** [openaudible/openaudible#514]
Please DO NOT close this issue until all Issue issues about this have been confirmed resolved.  This has not been fixed in Version 2.4.0.3beta.  I have posted comments about the unresolved in my own Issue and multiple other issues, but not once has anyone asked for me to test resolution.  The only thing you have done is close my Issue referencing other issues that you close without confirming resolution.  Just because an engineer tests the resolution does NOT indicate that the issue has been resolved.

I will provide you logs or any type of testing that you need.  All you have to do is ask.

**Comment by openaudible** [comment]
There is a beta that addresses this. 
See #510 for download link. 

As soon as we get some non-US feedback that it’s working we will make a full update. But it’s a big change so want to roll it out slower than usual.

**Comment by steve5053872** [comment]
I have installed it. It did not fix the issue.  Please read before you respond again.

**Comment by openaudible** [comment]
Sorry, missed that you are using the beta. 
What region are you in? Are you able to "Connect to Audible" from the Controls menu? It should say: Library Status: Connected: YES.  If not, some log data would be great. (Did you see the web page, did you enter your user/pass, pass the captcha?)

If connected, Controls Menu: Full Library Sync should get your library. If that is failing, some log messages would be nice. 

Logs are available from the File: Console window (CTRL-E)

**Comment by openaudible** [comment]
Also, OpenAudible can only backup your purchases. 
Free books, podcasts, and "all you can consume" type items (like Romance Series) are not something that can be backed up.

**Comment by openaudible** [comment]
Lastly, we just fixed our beta so that you can send a bug report from the About Menu. Just reference this or Issue #510. 
It should give us a good idea of what is going on if you send it after trying a "full sync library".

**Comment by oboe45** [comment]
Downloaded and installed latest Beta.  Still having difficulty connecting
to Audible.ca;   the app wants to log me into .US regardless of the drop
down option I select - I experimented and cursed and the only way I can
find to log into Audible.ca in the app is:
A. First log into the Audible.ca web site in any browser
B. Then log into the App, going through the CAPTCHa etc.   Make sure that
"save password" is on
C. You will appear to be logged into Audible US, so select the OpenAudible
Browser in the menu.  Close the Browser displayed and log in again to the
App browser to Audible.CA
D.  You may see now that you are switched to CA in the App, if not, logout
from the App and login again - with luck this time it remembers and sets
the drop down preference to CA.

Seems like a ton of effort just to get connected to the right Audible, but
it works, sometimes.

Could you fix this process?

**Comment by steve5053872** [comment]
I have updated to 2.4.0.4beta. A Full Library Scan still only finds 63 of
168 books.  I have tried to use the About > Send Bug Report..., but I get
the following error after clicking the Send button.  My anti Virus is
Norton 360.  I have disabled every option I can.

 [image: image.png]

Regards,
Steve Broyles
19070 E Creekside Dr
Parker, CO 80134

Cell    303 514-6220
Home 720 207-0384

**Comment by steve5053872** [comment]
I have updated to 2.4.0.4beta. A Full Library Scan **still only finds 63 of 168** books.  I have tried to use the About > Send Bug Report..., but I get the following error after clicking the Send button.  My anti-virus is Norton 360.  I have disabled every option I can, but I still get the following error:
![image](https://user-images.githubusercontent.com/67202338/103309781-42a57980-49d3-11eb-9b83-fb4eaa3eb100.png)

Is there any other way I can send you a bug report?

Steve

**Comment by LeeThompson** [comment]
Having a similar issue with 2.3.5 on Windows, only syncing 40 books; I have many more than that.

**Comment by openaudible** [comment]
Working on new version for non-us regions. 
If you’re in the US try the beta .4 and use the bug report if any problems. Will check bug report full error.

**Comment by steve5053872** [comment]
As I have posted and emailed, BUG REPORT DOES NOT WORK.  It would help if
you would read my replies.

Regards,
Steve Broyles

Mobile: (303) 514-6220

**Comment by openaudible** [comment]
@steve5053872 The "bug reporter full" means you tried to send a report too many times in one session... but didn't see any bugs from 2.4.0.4 from you, so not sure if you got other earlier errors. The only way to fix that is to restart the openaudible application.. try to do the library sync, then send a bug report.. and if you don't get a "success", take a look at the console (ctrl-E).

**Comment by openaudible** [comment]
Hi Steve. Bug reporter was definitely broken on the server. It should be fixed as of last night. (You may need to restart OA.)

**Comment by steve5053872** [comment]
I updated to OpenAudible_2.4.0.5_beta_win.exe and submitted a bug report.
Bug Report worked as expected.  Unfortunately, .5 beta did not resolve the
issue.  Let me know if there is anything else I can do to help.

Regards,
Steve Broyles

Cell    303 514-6220

**Comment by openaudible** [comment]
Hi @steve5053872 

Can you verify that the missing books are actual paid books, not podcasts, episodes, free items, romance series, all you can consume, etc? And also that they are still available on the audible page? Maybe provide a sample? We've seen some reports where podcasts show up in the list, but don't contain any download information. 

Posted a new version with some more logging.

**Comment by steve5053872** [comment]
Sorry for the slow response.  After talking with Audible, I figured out the
underlying issue.  I had a  pre-Amazon(PA) acquisition Audible account.
Several years ago, I merged the pre-Amazon library into my current Amazon
account.

The PA books were not showing up consistently.  The PA books were in my
total book *count* of my Amazon Audible library but did not show up when
paging through my Amazon Audible library.  Now, when I log-in to my Amazon
account, they are not showing in my book count or when I page through
the library.  I can, however, get to them by logging into the PA Audible
account.  This solves my real problem, in that I can get all of my books
downloaded and converted to my file format of choice.   From my
perspective, the issue is resolved. Thank you for all of your help and
patience.

Regards,
Steve
